src/quequations.jl

```diff
@@ -0,0 +1,53 @@
+abstract QuEquation
+
+immutable QuSchrodingerEq{H<:QuBase.AbstractQuMatrix} <: QuEquation
+    hamiltonian::H
+    QuSchrodingerEq(hamiltonian) = new(hamiltonian)
+end
+
+QuSchrodingerEq{H<:QuBase.AbstractQuMatrix}(hamiltonian::H) = QuSchrodingerEq{H}(hamiltonian)
+
+immutable QuLiouvillevonNeumannEq{H<:QuBase.AbstractQuMatrix} <: QuEquation
+    liouvillian::H
+    QuLiouvillevonNeumannEq(liouvillian) = new(liouvillian)
+end
+
+QuLiouvillevonNeumannEq{H<:QuBase.AbstractQuMatrix}(liouvillian::H) = QuLiouvillevonNeumannEq{H}(liouvillian)
+
+function liouvillian_op(h::QuBase.AbstractQuMatrix)
```

Comment:
This function will need some description (and comments), but we can do this later in connection with the Lindblad type.